src/components/DeploymentFormV2.vue

```diff
@@ -107,6 +119,18 @@
   const { validate } = useValidationObserver()
   const { errors, validate: validateParameters } = useSchemaValidation(schema, parameters)
   const { state: overrideState, error: overrideError } = useValidation(jobVariables, isJson('Job variables'))
+  const { state: nameState, error: nameError } = useValidation(name, (value) => {
+    if (props.mode === 'update') {
+      return true
+    }
```

Comment:
Doesn't this mean I could pass validation if I'm updating instead of duplicating and provide an invalid name?

Reply:
We disable the input for updating a deployment because a deployment name can't be updated

Reply:
ah yeah good point